ubiconfig/utils/config_schema.json

```diff
@@ -22,35 +22,38 @@
             "required": ["name", "stream"]
             }
     },
-    "content_sets": {
-        "debuginfo": {"$ref": "#/definitions/content_sets_base"},
-        "rpm": {"$ref": "#/definitions/content_sets_base"},
-        "srpm": {"$ref": "#/definitions/content_sets_base"}
-    },
-    "arches": {
-        "type": "array",
-        "items": {"type": "string"}
-    },
-    "modules": {
-        "type": "object",
-        "properties": {
-            "include": {
-                "type": "array",
-                "items":{"$ref": "#/definitions/module"}
-                }
-            }
+    "properties":{
+        "content_sets": {
```

Comment:
Is missing "type": "object" in here correct?  And shouldn't these debuginfo/rpm/srpm also be under "properties", or they have the same bug as is being fixed at the top-level here?

Reply:
whoops. I missed this.
Yeah, it will have the same bug the top level. I'll add it.